consensus/sender.py

```diff
@@ -9,10 +9,9 @@
 
 w3 = Web3(Web3.WebsocketProvider(config.INFURA_URL))
 db = ArangoClient().db('_system')
-nonce = w3.eth.getTransactionCount(config.ADDRESS)
 
 def sendTransaction(data):
-    global nonce
+    nonce = w3.eth.getTransactionCount(config.ADDRESS, 'pending')
```

Comment:
What does the 2nd argument `'pending'` do? I can't find it in the [documentation](https://web3py.readthedocs.io/en/stable/web3.eth.html#web3.eth.Eth.getTransactionCount).

Reply:
the second argument in the documentation named `block_identifier` which is not documented for this method but documented for some other methods with `latest`, `earliest`, `pending` valid values.
`pending` means consider `pending` transactions in response so if for example 10 transactions come together and all sent and still none of them are mined, each time we count the transaction, it counts the `pending` transactions. If not pass this, all the results will be same when we send lots of transactions together.